### PR TITLE
Bugfix 112477

### PR DESCRIPTION
**Pull request by andrasguseo**
Removed 'function_exists()'
Updated readme

### CODE REVIEW

**Review by cliffordp — CHANGES_REQUESTED** [review]
Let's instead use the `tribe_events_parse_query` _action_ hook and see if that works. That way, we won't need `remove_et_custom_posts_per_page()` to check if it's an events query, and also so it doesn't fire on every `parse_query` filter.

File: readme.txt [hunk 1]
```diff
@@ -35,6 +35,10 @@ We're always interested in your feedback and our [premium forums](https://theeve
 
 == Changelog ==
 
+= 1.2.1 2018-08-15 =
+
+* Fixed a bug due to which the post per page fix stopped working
```
Comment (cliffordp) [inline readme.txt]: Please clarify the wording a bit, such as
> Fix - Posts per page customization works again

(Not the best but you get the idea - simplify)

Comment (andrasguseo) [reply readme.txt]: I tried your suggestions for the action hook / filter, but didn't work for me, so leaving it for you.

**Comment by cliffordp** [comment]
Per Slack DM with Andras, I took over to finish this out.

**Pull request merged**